### ISSUE TITLE
Document app_agent_receiver API endpoints

### ISSUE THREAD

**Issue by Snapstromegon** [grafana/agent#5230]
### Request

Add detailed documentation on how a 3rd party developer can use the app_agent_receiver (https://grafana.com/docs/agent/latest/static/configuration/integrations/integrations-next/app-agent-receiver-config/) to submit measurements (and logs or traces) via the API as a client.

### Use case

The faro-web-sdk is huge (117kb) in my test build based on a rollup build with this JS:

```js
import { initializeFaro } from "@grafana/faro-web-sdk";

initializeFaro({
  app: {
    name: "hoeser.dev",
    version: "1.0.0",
  },
  url: "http://localhost:12347/collect",
});
```

To hit my site targets, I'm trying to keep my JS in total well below 100kb, so I've implemented a minimal version of faro https://github.com/Snapstromegon/hoeser.dev/blob/f37c660abb62f0b674574c3791abfb2f4ffa779f/assets/js/common/mini-faro.ts, that just implements the Web Vitals measurement and therefore is only ~9.2kb in size.

I think there are more users, who are looking for a small solution, even when it's less powerful (e.g. I don't care about user agents at all).

**Comment by tpaschalis** [comment]
Thanks for opening an issue. Just to understand, do you propose documenting the endpoints that the app_agent_receiver integration is using so that people can submit data to it without using Faro?

**Comment by Snapstromegon** [comment]
@tpaschalis Yes, exactly. This would be my ideal solution. This would allow 3rd party devs to know which fields in the API calls are required, what they mean and what values they can take.

**Comment by tpaschalis** [comment]
cc @rlankfo @kpelelis are these documented on the Faro side by some chance?

**Comment by rlankfo** [comment]
@tpaschalis sorry for the delayed response. @cedricziel started the process to document the faro receiver here: https://github.com/grafana/faro/pull/7
